ReactWindows/ReactNative.Shared/UIManager/NativeViewHierarchyManager.cs

```diff
@@ -55,7 +54,6 @@ namespace ReactNative.UIManager
     /// 
     /// TODO: 
     /// 1) AnimationRegistry
-    /// 2) ShowPopupMenu
```

Comment:
TODO is still relevant if you aren't going to implement it for WPF at the same time